test/start-kubernetes.sh

```diff
@@ -351,7 +351,27 @@ function init_kubernetes_cluster() (
     echo "Installing dependencies on cloud images, this process may take some minutes"
     vm_id=0
     pids=""
-    for ip in $(print_ips); do
+    IPS=$(print_ips)
+    # in Fedora-31 case, add 'systemd.unified_cgroup_hierarchy=0' to kernel cmdline and reboot
+    if [ "$TEST_DISTRO" = "fedora" -a "$TEST_DISTRO_VERSION" = "31" ]; then
+        for ip in ${IPS}; do
+            ssh $SSH_ARGS ${CLOUD_USER}@${ip} "sudo sed -i 's/n8\"/n8 systemd.unified_cgroup_hierarchy=0\"/g' /etc/default/grub"
```

Comment:
> should avoid two spell/outs of same string

I think copying the string is more readable and thus wouldn't use a variable.

> as 1st one is sed-line inside single quotes, how to use string there

This should work:
```
SYSTEMD_UNIFIED_CGROUP_HIERARCHY=systemd.unified_cgroup_hierarchy

ssh $SSH_ARGS ${CLOUD_USER}@${ip} "sudo sed -i 's/n8\"/n8 ${SYSTEMD_UNIFIED_CGROUP_HIERARCHY}=0\"/g' /etc/default/grub"
```

That's because the `${SYSTEMD_UNIFIED_CGROUP_HIERARCHY}` gets expanded as part of the outer double-quoted string, not as part of the inner single-quoted string.

If that wasn't the case, one could combine both inside a single parameter:
```
foo="   world   "
echo 'hello '"${foo}"', how are you?'
```